Makefile

```diff
@@ -21,289 +21,39 @@ endif
 
 include $(BOLOS_SDK)/Makefile.defines
 
-DEFINES_LIB = USE_LIB_ETHEREUM
-APP_LOAD_PARAMS = --curve secp256k1 $(COMMON_LOAD_PARAMS)
-# Allow the app to use path 45 for multi-sig (see BIP45).
-APP_LOAD_PARAMS += --path "45'"
-# Samsung temporary implementation for wallet ID on 0xda7aba5e/0xc1a551c5
-APP_LOAD_PARAMS += --path "1517992542'/1101353413'"
-
-##################
-# Define Version #
-##################
-
-APPVERSION_M = 1
-APPVERSION_N = 11
-APPVERSION_P = 0
-APPVERSION = $(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)-dev
-APP_LOAD_FLAGS = --appFlags 0xa40 --dep Ethereum:$(APPVERSION)
-
-#########################
-# Set Chain environment #
-#########################
+########################################
+#        Mandatory configuration       #
+########################################
 
 ifeq ($(CHAIN),)
     CHAIN = ethereum
+    APPNAME = Ethereum
```

Comment:
This should not be needed since it is already declared in `makefile_conf/chain/ethereum.mk`

Reply:
Yes, but a reason I didn't explain, using the VSCode extension, I cannot delete the App from the device if the name here is `ethereum`, whereas it works with `Ethereum`.

Reply:
Then it's a VS Code extension problem.

Reply:
Adding a comment to remember removing the APPNAME definition here, when VSCode extension will be updated